sql/core/src/main/scala/org/apache/spark/sql/sources/DataSourceStrategy.scala

```diff
@@ -179,13 +364,13 @@ private[sql] object DataSourceStrategy extends Strategy {
         translate(child).map(sources.Not)
 
       case expressions.StartsWith(a: Attribute, Literal(v: UTF8String, StringType)) =>
-        Some(sources.StringStartsWith(a.name, v.toString))
+        Some(sources.StringStartsWith(a.name, v.toString()))
```

Comment:
It's because `UTF8String.toString()` was defined with `()`, just want to keep consistent with the definition. Or we probably want remove `()` from the method definition?